.github/workflows/golangci-lint.yml

```diff
@@ -12,10 +12,10 @@ jobs:
     name: lint
     runs-on: ubuntu-latest
     steps:
-      - name: Install Go 1.18
-        uses: actions/setup-go@v3
+      - name: Set up Go
+        uses: actions/setup-go@v2
         with:
-          go-version: "1.18"
```

Comment:
Do we want to bump the version of lint being used as well?

Reply:
Sure, maybe a separate PR?

Reply:
Makes sense since it only affects the action